### PR TITLE
document files:troubleshoot-transfer-ownership

### PR DESCRIPTION
**Pull request by mrow4a**
closes https://github.com/owncloud/docs/issues/2794

### CODE REVIEW

**Review by phil-davis — APPROVED** [review]
I adjusted some sentences, and squashed/rebased. My IDE automagically deleted whitespace from the end of lines, so there are a few morel little diffs.

@mmattel please also review if you have time.

**Comment by mmattel** [comment]
I have made a bunch of changes. All the stuff has been checked running an antora build and it looks fine.

**Review by mmattel — APPROVED** [review]
I of course approve your PR and my own changes :smile:

**Comment by mmattel** [comment]
Pls tell to which brances this needs a backport

File: modules/admin_manual/pages/configuration/server/occ_command/file_commands.adoc [hunk 1]
```diff
@@ -230,30 +236,109 @@ TIP: If many users are affected, it could be convenient to create a shell script
...
+  It will be moved into the destination user's `files` directory, into a directory name which follows the format: `transferred from <source-user> on <timestamp>`.
   Using the example above, it will be stored under: `data/<destination-user>/files/transferred from <source-user> on 20170426_124510/`
-. Currently file versions can't be transferred. 
+. Currently file versions can't be transferred.
   Only the latest version of moved files will appear in the destination user's account.
+
+== The files:troubleshoot-transfer-ownership command
+
+This command is used to scan for problems, that might have occurred during a run of
+ownership transfer using the above command set `files:transfer-ownership`.
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/server/occ_command/file_commands.adoc:299]:
```suggestion
ownership transfer using the above command `files:transfer-ownership`.
```

I don't understand why the word "set" is there.

**Review by phil-davis — COMMENTED** [review]
@mmattel just one tiny thing that I am not sure about.

**Comment by phil-davis** [comment]
> Pls tell to which brances this needs a backport

This command is to be released in 10.6.0 - so no backport needed. (Unless we want to try and manually "backport" the other changes made here that are not directly about the new command)

File: modules/admin_manual/pages/configuration/server/occ_command/file_commands.adoc (hunk 1 above)
Comment (mmattel) [reply modules/admin_manual/pages/configuration/server/occ_command/file_commands.adoc:299]: Thanks for catching it

**Review by phil-davis — APPROVED** [review]

**Pull request merged**